homeassistant/components/squeezebox/media_player.py

```diff
@@ -241,14 +224,18 @@ class SqueezeBoxEntity(MediaPlayerEntity):
     _last_update: datetime | None = None
     _attr_available = True
 
-    def __init__(self, player):
+    def __init__(self, player, server):
         """Initialize the SqueezeBox device."""
         self._player = player
+        self._server = server
         self._query_result = {}
         self._remove_dispatcher = None
         self._attr_unique_id = format_mac(player.player_id)
         self._attr_device_info = DeviceInfo(
-            identifiers={(DOMAIN, self._attr_unique_id)}, name=player.name
+            identifiers={(DOMAIN, self._attr_unique_id)},
+            name=player.name,
+            via_device=(DOMAIN, server.uuid),
```

Comment:
Is this device already registered?

Reply:
The media_player? of the server? the server will come if following commits

Reply:
If we don't have it now, let's omit it